demo/deploy-bookbuyer.sh

```diff
@@ -74,6 +74,14 @@ spec:
             - name: "CI_SLEEP_BETWEEN_REQUESTS_SECONDS"
               value: "$CI_SLEEP_BETWEEN_REQUESTS_SECONDS"
 
+      # This is used for Multicluster environment demos,
+      # where the bookstore will try to resolve these 2 hosts.
+      hostAliases:
+      - ip: 127.0.0.1
```

Comment:
Does this only work for Kind demo?

Reply:
It will work on any compliant Kubernetes distro.